### PR TITLE
Render nextAtom received from server

### PR DESCRIPTION
**Pull request by rpiaggio**
Render `nextAtom` steps from server state.

Some advances and cleanup regarding row styling.

### CODE REVIEW

File: modules/web/client/src/main/scala/observe/ui/components/sequence/SequenceTables.scala [hunk 1]
```diff
@@ -50,27 +53,38 @@ sealed trait SequenceTables[S, D](
...
-  protected[sequence] lazy val scienceSteps: List[SequenceRow.FutureStep[D]] =
-    config.science.map(steps).orEmpty
-
-  protected[sequence] lazy val acquisitionNextAtomId: Option[Atom.Id] =
-    config.acquisition.map(_.nextAtom.id)
+  private def steps(sequence: ExecutionSequence[D]): List[SequenceRow.FutureStep[DynamicConfig]] =
+    SequenceRow.FutureStep
+      .fromAtoms(
+        sequence.possibleFuture,
+        _ => none // TODO Pass signal to noise
```
Comment (cquiroz) [inline modules/web/client/src/main/scala/observe/ui/components/sequence/SequenceTables.scala:60]: WDYM? to display S/N?

File: modules/web/client/src/main/scala/observe/ui/components/sequence/steps/CurrentAtomStepRow.scala [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  breakpoint:        Breakpoint,
+  val isFirstOfAtom: Boolean
+) extends SequenceRow[DynamicConfig](
+      id = step.id.asRight,
+      instrumentConfig = step.instConfig.some,
+      stepConfig = step.stepConfig.some,
+      breakpoint = breakpoint,
+      isFinished = step.status.isFinished,
+      // TODO This could be an estimate for pending steps, or the time it took for finished steps.
+      // In either case, we don't have the information from the
```
Comment (cquiroz) [inline modules/web/client/src/main/scala/observe/ui/components/sequence/steps/CurrentAtomStepRow.scala:24]: `from the....`

**Review by cquiroz — APPROVED** [review]
LGTM

File: modules/web/client/src/main/scala/observe/ui/components/sequence/SequenceTables.scala (hunk 1 above)
Comment (rpiaggio) [reply modules/web/client/src/main/scala/observe/ui/components/sequence/SequenceTables.scala:60]: Yup, we are not displaying it ATM.

**Pull request merged**